src/main/java/fr/inria/diversify/automaticbuilder/GradleAutomaticBuilder.java

```diff
@@ -167,4 +233,57 @@ private String getPrintClasspathTask() {
                 "}";
     }
 
+    private String getPitTask() {
+        return getPitTask(null);
+    }
+
+
+    private String getPitTask(CtType<?> testClass) {
+        return getPitTaskConfiguration() + getPitTaskOptions(testClass);
+    }
+
+    private String getPitTaskConfiguration() {
+        return NEW_LINE + NEW_LINE + "buildscript {" + NEW_LINE +
+                "    repositories {" + NEW_LINE +
```

Comment:
You're right, I need to take care about this: if a buildscript section already exists, the injection should consider it. If you agree, it's matter of the next change (new branch :-)).